yivi-css/src/mixins/fonts.scss

```diff
@@ -15,6 +15,7 @@
 // Styleguide Mixins.fonts
 
 @import '../variables/fonts', '../variables/colours';
+@import '../../node_modules/npm-font-open-sans/open-sans';
```

Comment:
There is an issue with the build pipeline. Could you try to fix that first?

How does vendoring open sans affect the bundle size of `irma-frontend`? In earlier versions I removed the vendored fonts, because it led to a bundle size of multiple MBs. If that's the case too now, then I'd prefer the approach to not vender the fonts and recommend users to import open sans if they want. If they don't want that, then the fallback fonts are used automatically.

Reply:
Done